packages/parser/src/index.ts

```diff
@@ -5,4 +5,5 @@ export {
 } from '@typescript-eslint/typescript-estree';
 
 // note - cannot migrate this to an import statement because it will make TSC copy the package.json to the dist folder
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 export const version: string = require('../package.json').version;
```

Comment:
perhaps we should add a global type declaration to the project.
Something like
```ts
declare module '../package.json' {
  export const version = string;
}
```
would that fix this problem?

Reply:
I don't think so 😕 , in `@types/node/globals.d.ts` `NodeJS.Require` is typed as returning `any` from its `(id: string)` function signature.

```ts
interface Require {
  (id: string): any;
```